pandas/io/formats/style.py

```diff
@@ -72,43 +72,44 @@
     from matplotlib.colors import Colormap
 
 try:
-    import matplotlib as mpl
-    import matplotlib.pyplot as plt
+    import matplotlib as mpls
+    import matplotlib.pyplot as plts
```

Comment:
how about if, instead of changing to non-standard aliases, we change

https://github.com/pandas-dev/pandas/blob/97edd8e4ff89b4a3eedebd03bd718048fef339a3/pandas/io/formats/style.py#L3600

to use, say, `as (_pyplot, _matplotlib)` (and then rename accordingly)?

Reply:
Should be updated as of: a7d5fdee89263aa7a67a704bf5aa9b5b37b21bb5